### PR TITLE
gfx_draw_string_left_wrapped overload added

### PR DESCRIPTION
**Pull request by sohiladwani**
Planning to replace more calls just want to make sure I did the first correct.

### CODE REVIEW

File: src/openrct2-ui/windows/EditorObjectiveOptions.cpp [hunk 1]
```diff
@@ -987,7 +987,11 @@ static void window_editor_objective_options_main_paint(rct_window* w, rct_drawpi
     ft = Formatter::Common();
     ft.Add<rct_string_id>(STR_STRING);
     ft.Add<const char*>(gS6Info.details);
-    gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, x, y, width, STR_BLACK_STRING, COLOUR_BLACK);
+    ScreenCoordsXY screenCoords;
+    screenCoords.x = x;
+    screenCoords.y = y;
+    gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, screenCoords, width, STR_BLACK_STRING, COLOUR_BLACK);
+
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/EditorObjectiveOptions.cpp:994]:
```suggestion
    gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, { x, y }, width, STR_BLACK_STRING, COLOUR_BLACK);
```

You can create the objects in place following my suggestion. This function will probably have a `screenCoords` at some point, but then it will need to be used for all calls and you probably don't want to tackle it now.

File: src/openrct2/drawing/Text.cpp [hunk 2]
```diff
@@ -229,6 +229,26 @@ int32_t gfx_draw_string_left_wrapped(
     return layout.GetHeight();
```
Comment (tupaschoal) [inline src/openrct2/drawing/Text.cpp:229]: Make the old function call the new one to avoid code duplication

File: src/openrct2/drawing/Text.h [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/*****************************************************************************
```
Comment (tupaschoal) [inline src/openrct2/drawing/Text.h:1]: Please revert this change

File: src/openrct2/drawing/Text.cpp (hunk 2 above)
Comment (tupaschoal) [reply src/openrct2/drawing/Text.cpp:229]: Please make the old function call the new one, like I said, to avoid code duplication

Comment (sohiladwani) [reply src/openrct2/drawing/Text.cpp:229]: Sorry missed this, will do in next commit!
